backend/tests/services/knowledge/group_kb/test_group_kb_inheritance.py

```
@@ -0,0 +1,136 @@
+    test_group_owner,
+    test_subgroup,
+)
+
+
+class TestPermissionInheritance:
+    """Test permission inheritance from parent groups to subgroups."""
+
+    def test_owner_inherits_to_subgroup(
+        self,
+        test_db: Session,
+        test_subgroup,
```

Ruff (0.15.2)

[error] Redefinition of unused `test_subgroup` from line 30: `test_subgroup` redefined here

(F811)